src/components/CountryList/index.js

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import CountryItem from '../CountryItem';
+import CountryDetail from '../CountryDetail';
+
+export default function CountryList({ id }) {
+  const { countries, countryById, countriesBySearch } = useSelector(
+    (state) => state.countries,
+  );
+  if (countriesBySearch.length || (!id && !countryById?.id)) {
+    return (
+      <section className="countries">
+        {countriesBySearch.map((country) => (
+          <CountryItem key={country.id} country={country} />
+        ))}
+        {countriesBySearch.length === 0
+          && countries.map((country) => (
+            <CountryItem key={country.id} country={country} />
+          ))}
+      </section>
+    );
+  }
+  return (
+    <section className="countries">
+      {countryById.id && <CountryDetail country={countryById} />}
+    </section>
```

Comment:
- [ ] Kindly style the countries on the homepage to follow the design we have in the design. The design has alternating color patterns, but yours does not. Kindly add that.

  Your Implementation | Design guideline
  :---:|:---:
  ![udjs](https://user-images.githubusercontent.com/54848363/233783312-df3a0e8b-1474-4f8f-aa8f-c7d9302cc684.jpg) | ![uiww](https://user-images.githubusercontent.com/54848363/233783311-3c255ee9-6a28-484e-b2b8-b98f8343d799.jpg)